homeassistant/components/logger/__init__.py

```diff
@@ -36,6 +36,8 @@
 SERVICE_SET_DEFAULT_LEVEL_SCHEMA = vol.Schema({ATTR_LEVEL: _VALID_LOG_LEVEL})
 SERVICE_SET_LEVEL_SCHEMA = vol.Schema({cv.string: _VALID_LOG_LEVEL})
 
+EVENT_LOGGING_CHANGED = "logging_changed"
+
 CONFIG_SCHEMA = vol.Schema(
```

Comment:
We should update the `CONFIG_SCHEMA` to specify integrations to log. It would then take all loggers of that integration and apply the level.

Reply:
When we apply an integration level (either via config or via WS) we should store the old levels such that in the frontend we can enable turning off logging for an integration.